src/constants/theme/styles.ts

```diff
@@ -0,0 +1,6 @@
+export const STYLES = {
+  opacity: {
+    min: 0.2,
+    max: 1,
```

Comment:
If these are and will only be used for press opacity of buttons the naming should resemble that imo.

Reply:
renamed `pressOpacity`